### ISSUE TITLE
Enable more tests for the k8s-e2e-external-storage job

### ISSUE THREAD

**Issue by nixpanic** [ceph/ceph-csi#2015]
The `ci/centos/k8s-e2e-external-storage` runs the [Kubernetes end-to-end external storage tests](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external), based on the [different `driver-*.yaml` files](https://github.com/ceph/ceph-csi/tree/devel/scripts/k8s-storage). These driver files describe the capabilities of the provisioner.

Currently many features are not enabled in the drivers-test configuration. We ate aiming to have have all features that the provisioners support tested by the Kubernetes external storage tests. Each option that gets enabled should ideally be submitted by its own PR, referencing this Issue, use:

    Updates: #2015

**Comment by nixpanic** [comment]
_Edit: This comment has been moved into its own issue: https://github.com/ceph/ceph-csi/issues/2017#issue-865024313_

**Comment by Yuggupta27** [comment]
Currently busy with other features, not working on this atm.

**Comment by humblec** [comment]
Removing this from the 3.5 milestone tracker.

**Comment by yati1998** [comment]
Thanks, @Rakshith-R for assigning, will work on it.

**Comment by humblec** [comment]
@yati1998 shall we remove this from the release 3.6 tracker ?

**Comment by yati1998** [comment]
yes.

**Comment by humblec** [comment]
@yati1998 is this something we are making progress ? if yes, please feel free to link , I was looking into this partially and PRs like #3156 were a start, however I am doubtful that the pending tests will be able to bring up soon on at 3.7 which is planned at end of this month. Unless there is something which you are already working on, I would like to propose to move this out of 3.7.

**Comment by yati1998** [comment]
Yes, I am working on this by side, but facing issues to run tests locally. Will try to get something for 3.7, but still not sure. I will link once I have something. Hope that's fine.

**Comment by humblec** [comment]
Sure, please adjust the tracker accordingly, we have a week more atleast.
